systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -1081,212 +1129,193 @@ void testLabelModificationDoesNotBreakCluster(ExtensionContext extensionContext)
             .withMessageCount(testStorage.getMessageCount())
             .build();
 
-        resourceManager.createResource(extensionContext,
-            kafkaClients.producerStrimzi(),
-            kafkaClients.consumerStrimzi()
-        );
-        ClientUtils.waitForClientsSuccess(testStorage);
-
-        Map<String, String> zkPods = PodUtils.podSnapshot(testStorage.getNamespaceName(), testStorage.getZookeeperSelector());
-        Map<String, String> kafkaPods = PodUtils.podSnapshot(testStorage.getNamespaceName(), testStorage.getKafkaSelector());
+        LOGGER.info("--> Test Strimzi related expected labels of managed kubernetes resources <--");
 
-        LOGGER.info("Waiting for kafka stateful set labels changed {}", labels);
-        StrimziPodSetUtils.waitForStrimziPodSetLabelsChange(testStorage.getNamespaceName(), testStorage.getKafkaStatefulSetName(), labels);
+        LOGGER.info("---> PODS <---");
 
-        LOGGER.info("Getting labels from stateful set resource");
-        Map<String, String> kafkaLabels = StrimziPodSetUtils.getLabelsOfStrimziPodSet(testStorage.getNamespaceName(), testStorage.getKafkaStatefulSetName());
+        List<Pod> pods = kubeClient(testStorage.getNamespaceName()).listPodsByPrefixInName(testStorage.getNamespaceName(), testStorage.getClusterName());
 
-        LOGGER.info("Verifying default labels in the Kafka CR");
+        for (Pod pod : pods) {
+            LOGGER.info("Getting labels from {} pod", pod.getMetadata().getName());
```

Comment:
```suggestion
            LOGGER.info("Verify labels of pod/{}", pod.getMetadata().getName());
```

maybe you can also print namespace